assets/css/slides-rdmbites.css

```diff
@@ -5,7 +5,7 @@
 	right: 1em;
 }
 
-.remark-slide-content h1 { font-size: 3.5rem; }
-.remark-slide-content h2 { font-size: 3rem; }
-.remark-slide-content h3 { font-size: 2.5rem; }
-p {font-size: 1.8rem;}
+.remark-slide-content h1 { font-size: 3.5rem; font-family: Calibri, sans-serif;}
+.remark-slide-content h2 { font-size: 3rem; font-family: Calibri, sans-serif;}
```

Comment:
Hmm, I'm not sure what to do here. I suspect that renders fine for you, but it won't for me and it won't for the servers rendering the video. And that font isn't openly licensed so I'm not sure how to get it on linux. (There's an asterisk here, that font licensing is quite weird in most parts of the world but it still feels dodgy.)

I've seen one person suggesting replacing Calibri with Carlito which has relatively minor differences (except &/$) would that be an option here or so?
http://www.identifont.com/differences?first=Carlito&second=Calibri